Assets/BossRoom/Scripts/Shared/Net/ConnectionManagement/GameNetPortal.cs

```diff
@@ -214,19 +202,15 @@ public async void StartUnityRelayHost(CancellationToken cancellationToken)
 
                     try
                     {
-                        await UnityServices.InitializeAsync();
-                        if (!AuthenticationService.Instance.IsSignedIn)
-                        {
-                            await AuthenticationService.Instance.SignInAnonymouslyAsync();
-                            var playerId = AuthenticationService.Instance.PlayerId;
-                            Debug.Log(playerId);
-                        }
-
                         // we now need to get the joinCode?
                         var serverRelayUtilityTask = UnityRelayUtilities.AllocateRelayServerAndGetJoinCode(k_MaxUnityRelayConnections);
                         await serverRelayUtilityTask;
                         // we now have the info from the relay service
-                        var (ipv4Address, port, allocationIdBytes, connectionData, key, _) = serverRelayUtilityTask.Result;
+                        var (ipv4Address, port, allocationIdBytes, connectionData, key, joinCode) = serverRelayUtilityTask.Result;
+
+                        m_LocalLobby.RelayJoinCode = joinCode;
+                        //next line enabled lobby and relay services integration
```

Comment:
next line was confusing and I had to dig deeper in the doc to understand what this was doing. Correct me if I'm wrong, but you can have clients set some random data + allocation ID themselves (else lobby data is only host writable).
the "+ allocation ID" is where the magic happens for automatically handling connection, is that it?